client/reader/site-stream/featured.jsx

```diff
@@ -23,8 +25,35 @@ export default React.createClass( {
 
 	getStateFromStores( store ) {
 		store = store || this.props.store;
+
+		let postKeys = store.get();
+		let posts = postKeys.map( postKey => {
+			let post = FeedPostStore.get( postKey );
+			if ( ! post || post._state === 'minimal' ) {
+				FeedPostStoreActions.fetchPost( postKey );
+				post._state = 'pending';
```

Comment:
Posts are supposed to be immutable, so setting this here isn't the best. Probably want to set it as part of the fetchPost action, or teach the rest of the code that `minimal` is effectively `pending`.